tensornetwork/block_sparse/blocksparsetensor.py

```diff
@@ -391,6 +391,13 @@ def __rmul__(self, number: np.number) -> "ChargeArray":
   def __truediv__(self, number: np.number) -> "ChargeArray":
     raise NotImplementedError("__truediv__ not implemented for ChargeArray")
 
+  def item(self):
+    if self.ndim == 0:
+      if len(self.data) == 1:
+        return self.data[0]
+      return self.dtype.type(0.0)  #the default value is currently 0.0
```

Comment:
Why do we have a default here?

Reply:
that's the default value for tensor elements that do not conserve charge. In fact, I think that this line should never be hit, so I'll remove it